tutorial10part1.py

```diff
@@ -0,0 +1,57 @@
+import pandas
+import numpy
+import scipy
+import scipy.integrate as spint
+from plotnine import *
+
+r_list=[-0.1,0.1,0.4,0.8,1]
+
+def growthRate(y,to,r,K):
+    N=y[0]
+    dNdt=r* (1-N/K)* N
+
+    return [dNdt]
+
+for i in r_list:
+    params=(i,100)
+    N0=10
+    times=range(0,1000)
+    
+modelSim=spint.odeint(func=growthRate,y0=N0,t=times,args=params)
```

Comment:
Add modelSim and store the output in a dataframe into the loop. So you can store and print out: 

rs=[-0.1,0.1,0.4,0.8,1]
store_rs=pandas.DataFrame({"time":times,"r1":0,"r2":0,"r3":0,"r4":0,"r5":0})

for i in range(0,len(rs)):
    pars=(rs[i],K)
    sim=si.odeint(func=ddSim,y0=y0,t=times,args=pars)    
    store_rs.iloc[:,i]=sim[:,0]

ggplot(store_rs,aes(x="time",y="r1",color="-0.1"))+geom_line()+geom_line(aes(x="time",y="r2",color="0.1"))+geom_line(aes(x="time",y="r3",color="0.4"))+geom_line(aes(x="time",y="r4",color="0.8"))+geom_line(aes(x="time",y="r5",color="1.0"))+theme_classic()+labs(x="time",y="N")